mlflow/llama_index/__init__.py

```diff
@@ -334,14 +334,14 @@ def load_model(model_uri, dst_path=None):
     settings_path = os.path.join(local_model_path, _SETTINGS_FILE)
     # NB: Settings is a singleton and can be loaded via llama_index.core.Settings
     deserialize_settings(settings_path)
-
     return _load_index(local_model_path, flavor_conf)
 
 
 def _load_pyfunc(path, model_config: Optional[Dict[str, Any]] = None):
+    index = load_model(path)
```

Comment:
This solves the issue #1. We should load the index via `load_model()` that includes `deserialize_setting` call, rather than `_load_index()`.